internal/criocli/criocli.go

```diff
@@ -257,66 +334,76 @@ func mergeConfig(config *libconfig.Config, ctx *cli.Context) error {
 		config.AddInheritableCapabilities = ctx.Bool("add-inheritable-capabilities")
 	}
 
-	if ctx.IsSet("default-sysctls") {
-		config.DefaultSysctls = StringSliceTrySplit(ctx, "default-sysctls")
+	if ctx.IsSet("uid-mappings") {
+		config.UIDMappings = ctx.String("uid-mappings")
 	}
 
-	if ctx.IsSet("default-ulimits") {
-		config.DefaultUlimits = StringSliceTrySplit(ctx, "default-ulimits")
+	if ctx.IsSet("minimum-mappable-uid") {
+		config.MinimumMappableUID = ctx.Int64("minimum-mappable-uid")
 	}
 
-	if ctx.IsSet("pids-limit") {
-		config.PidsLimit = ctx.Int64("pids-limit")
+	if ctx.IsSet("gid-mappings") {
+		config.GIDMappings = ctx.String("gid-mappings")
 	}
 
-	if ctx.IsSet("log-size-max") {
-		config.LogSizeMax = ctx.Int64("log-size-max")
+	if ctx.IsSet("minimum-mappable-gid") {
+		config.MinimumMappableGID = ctx.Int64("minimum-mappable-gid")
 	}
+}
 
-	if ctx.IsSet("log-journald") {
-		config.LogToJournald = ctx.Bool("log-journald")
+func mergeCgroupConfig(config *libconfig.Config, ctx *cli.Context) {
+	if ctx.IsSet("cgroup-manager") {
+		config.CgroupManagerName = ctx.String("cgroup-manager")
 	}
 
-	if ctx.IsSet("cni-default-network") {
-		config.CNIDefaultNetwork = ctx.String("cni-default-network")
+	if ctx.IsSet("blockio-config-file") {
+		config.BlockIOConfigFile = ctx.String("blockio-config-file")
 	}
 
-	if ctx.IsSet("cni-config-dir") {
-		config.NetworkDir = ctx.String("cni-config-dir")
+	if ctx.IsSet("blockio-reload") {
+		config.BlockIOReload = ctx.Bool("blockio-reload")
 	}
 
-	if ctx.IsSet("cni-plugin-dir") {
-		config.PluginDirs = StringSliceTrySplit(ctx, "cni-plugin-dir")
+	if ctx.IsSet("irqbalance-config-file") {
+		config.IrqBalanceConfigFile = ctx.String("irqbalance-config-file")
 	}
 
-	if ctx.IsSet("image-volumes") {
-		config.ImageVolumes = libconfig.ImageVolumesType(ctx.String("image-volumes"))
+	if ctx.IsSet("irqbalance-config-restore-file") {
+		config.IrqBalanceConfigRestoreFile = ctx.String("irqbalance-config-restore-file")
 	}
```

Comment:
I don't think they are related to cgroup.

Reply:
Yeah that should be misc.